model/src/testFixtures/proto/spine/validation/require_option_spec.proto

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2025, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+syntax = "proto3";
+
+package spine.validation;
+
+import "spine/options.proto";
+
+option (type_url_prefix) = "type.spine.io";
+option java_package = "io.spine.validation";
+option java_outer_classname = "RequireOptionSpecProto";
+option java_multiple_files = true;
+
+import "google/protobuf/timestamp.proto";
+
+// Declares the `(require)` option with the inapplicable `bool_field`.
+message RequireWithBool {
+    option (require).fields = "string_field | bool_field | timestamp_field";
+
+    string string_field = 1;
+    bool bool_field = 2;
+    google.protobuf.Timestamp timestamp_field = 3;
+}
+
+// Declares the `(require)` option with the inapplicable `int32_field`.
+message RequireWithSignedInt {
+    option (require).fields = "string_field | timestamp_field | int32_field";
+
+    string string_field = 1;
+    int32 int32_field = 2;
+    google.protobuf.Timestamp timestamp_field = 3;
+}
+
+// Declares the `(require)` option with the inapplicable `uint64_field`.
+message RequireWithUnsignedInt {
+    option (require).fields = "string_field | uint64_field";
+
+    string string_field = 1;
+    uint64 uint64_field = 2;
+    google.protobuf.Timestamp timestamp_field = 3;
+}
+
+// Declares the `(require)` option with the inapplicable `double_field`.
+message RequireWithDouble {
+    option (require).fields = "double_field";
```

Comment:
I can see the test case now that I've mentioned earlier. Please disregard that comment.